StripeFinancialConnections/StripeFinancialConnections/Source/Native/PartnerAuth/PrepaneImageView.swift

```diff
@@ -12,41 +12,78 @@ import WebKit
 
 final class PrepaneImageView: UIView {
 
-    init(imageURLString: String) {
-        super.init(frame: .zero)
-        backgroundColor = .backgroundContainer
-        clipsToBounds = true
-        layer.cornerRadius = 8.0
+    private let centeringView: UIView
 
+    init(imageURLString: String) {
         // first we load an image (or GIF) into a WebView
         let imageView = GIFImageView(gifUrlString: imageURLString)
         // the WebView is surrounded by a background that imitates the GIF presented inside of a phone
         let phoneBackgroundView = CreatePhoneBackgroundView(imageView: imageView)
         // we center the phone+gif in the middle
         let centeringView = CreateCenteringView(centeredView: phoneBackgroundView)
-        addAndPinSubview(centeringView)
+        self.centeringView = centeringView
+        super.init(frame: .zero)
+
+        // background color of the whole view
+        centeringView.backgroundColor = .backgroundContainer
+
+        addAndPinSubview(
+            centeringView,
+            // the insets expand the view beyond the bounds
+            // to stretch the `PrepaneImageView` for the
+            // full width of the pane
+            insets: NSDirectionalEdgeInsets(
+                top: 0,
+                leading: -Constants.Layout.defaultHorizontalMargin,
+                bottom: 0,
+                trailing: -Constants.Layout.defaultHorizontalMargin
+            )
+        )
     }
 
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+
+    override func layoutSubviews() {
+        super.layoutSubviews()
+
+        // Clip-to-bounds the top and bottom, but leave
+        // left/right unclipped.
+        //
+        // This code is like setting `clipsToBounds = true`,
+        // except for just top/bottom.
+        let path = UIBezierPath(
+            rect: CGRect(
+                x: (bounds.width - centeringView.frame.width) / 2,
+                y: 0,
+                width: centeringView.frame.width,
+                height: bounds.height
+            )
+        ).cgPath
+        let maskLayer = CAShapeLayer()
+        maskLayer.path = path
+        layer.mask = maskLayer
```

Comment:
does this work out ~OK (doesn't crop too much) if the device is in landscape?

Reply:
yea rotations work as expected (that being said, and maybe you are alluding to this, due to _unrelated_ logic, because theres a notch, and the "pane" (/view) is bound to safe area insets, there is a gap)